samples/TR/index.html

```diff
@@ -87,13 +87,8 @@
 	The first one (“Classic”) shows it in its usual form.
 	The second one (“Vivliostyle”) shows the same document
 	in a paginated yet responsive view
+	(spread view if window is wide)
```

Comment:
Sorry, this pull req was so early. I intended this should be merged after spread_auto is finished.